nix/pkgs/thorp/default.nix

```diff
@@ -0,0 +1,39 @@
+{ pkgs, sources }:
+let
+  mvn2nix = import sources.mvn2nix { };
```

Comment:
I would probably move mvn2nix into `pkgs/default.nix` and pass it in as dependency to this.